crypto/x509/x509_test.cc

```diff
@@ -3190,9 +3190,11 @@ TEST(X509Test, PrettyPrintIntegers) {
       "-42",
       "256",
       "-256",
+      "4886718345",
+      "-4886718345",
       // Large numbers are pretty-printed in hex to avoid taking quadratic time.
-      "0x0123456789",
-      "-0x0123456789",
+      "0x0123456789012345678901234567890123456789",
+      "-0x0123456789012345678901234567890123456789",
```

Comment:
minor nit: you should only need 32 hex digits to hit your threshold of 2^128

Reply:
True, I just wanted the numbers to align. Will change